src/job_handler_plugins/radix/__init__.py

```diff
@@ -16,6 +16,12 @@ def _get_job_url(job: Job) -> str:
     return f"http://{job_name}:{scheduler_port}/api/v1/jobs"
 
 
+def list_of_env_to_dict(env_vars) -> dict:
+    keys = [s.split("=")[0] for s in env_vars]
+    values = [s.split("=")[1] for s in env_vars]
+    return dict(zip(keys, values))
```

Comment:
```suggestion
def list_of_env_to_dict(env_vars) -> dict:
new_dict = { keys, values for s.split("=",1) in env_vars} 
```
Should be possible to write this a bit cleaner with dict comprehension

Reply:
I guess it should be something more like
`new_dict = { s.split("=")[0], s.split("=")[1]  for s in env_vars}`?